.bazelrc

```diff
@@ -1,3 +1,3 @@
 # We can't create a bzl_library for rules-swift because of its visibility,
 # so circumvent by not using the sandbox
-build --strategy=Stardoc=standalone
+common --strategy=Stardoc=standalone
```

Comment:
I see the strategy option is documented in the bazel manuel and includes a valid value of standalone. But why is Stardoc also included here?

Reply:
because we only want to run actions with `Stardoc` as the mnemonic in standalone mode (we don't want to run other actions in standalone mode)